source/content/branch.md

```diff
@@ -0,0 +1,60 @@
+---
+title:  Automated testing and deployment with Branch
+description: Learn how to use Branch to automatically build, test and deploy to Pantheon from a third party Git provider, like GitHub, GitLab or Bitbucket.
+categories: [automate]
+tags: [ci, git, workflow]
+contributors: [petersuhm]
+---
+
+[Branch](https://www.branchci.com/) is an automated deployment tool, designed to help you automate the building, testing and deployment of your WordPress projects whenever you push to Git.
+
+Branch makes it easy to deploy your code to Pantheon from GitHub, Bitbucket or Gitlab. Simply connect Branch to your Pantheon account with a machine token and use the built-in Pantheon recipes for Git Push, rsync or MultiDev deployments.
+
+## Before You Begin
+- If you don't already have a Branch account, [create one now](https://www.branchci.com/).
+- Manage and store your site's code repository on a third party hosting service, such as [GitHub](/guides/collaborative-development).
+
+## Deploy GitHub to Pantheon with Branch
+
+### Connect Your GitHub Repository
+1. If this is the first time you log into Branch, you will see a list of all your Git repositories. If you are an existing user, click **Add new** in the sidebar to see the same list.
```

Comment:
My experience was that the first time you log in you're presented with a setup wizard. Should we document that?

Reply:
The first part of the setup wizard should be to pick from a list of all your Git repositories, so you can connect one of them to your project. Was this not the experience you had?